env/gpu/py3-master.yml

```diff
@@ -27,7 +27,7 @@ dependencies:
   - regex
   - pip:
     - pylint-quotes<0.2
-    - mxnet-cu92mkl>=1.5.0b20190407
+    - mxnet-cu101mkl>=1.5.0
```

Comment:
this should be a nightly version.

Reply:
Ie the `master` in `py3-master.yml` refers to MXNet master branch?

Reply:
yes